src/components/NcRichContenteditable/NcRichContenteditable.vue

```diff
@@ -47,16 +47,26 @@ This component displays contenteditable div with automated `@` [at] autocompleti
 			:user-data="userData"
 			placeholder="Try mentioning user @Test01 or inserting emoji :smile"
 			@submit="onSubmit" />
-		<br>
-		<br>
+
+		<h5>Output - default</h5>
+		{{ message }}
```

Comment:
I'd add `white-space: pre-wrap` here to display new lines.

Reply:
It will be nice, but white spaces are collapsed [by default](https://developer.mozilla.org/en-US/docs/Web/CSS/white-space), we have `pre-wrap` only for NcRichText
I wouldn't confuse other developers on that place (or we could show both options)

Reply:
pre-wrap looks weird here, I'd like to not add it for docs

![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/93392545/78c8117f-acdf-4570-a19b-65de8ad6212c)

Reply:
> Do you mean docs block in NcRichText file?

Yep, to have more Markdown examples in the docs like we have in tests and bugs. To see all the "unobvious" edge cases.